less/navigation.less

```diff
@@ -1 +1,44 @@
-// Navigation code here
\ No newline at end of file
+// Navigation code here
+
+.navsection{
+    display: flex;
+    flex-direction: column;
+    background: #F0F0F5; 
+    position: fixed;
+    top: 0;
+    width: 100%;
```

Comment:
Sticky-Nav! Try adding opacity on the scroll down so you can still see the page.